src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Extensions/CollectionExpressionExtensions.cs

```diff
@@ -0,0 +1,178 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Immutable;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+
+namespace Microsoft.CodeAnalysis.Shared.Extensions;
```

Comment:
It's a bit strange this is named "Extensions" and not one method is an extension method.

Reply:
fair.  i'll call them utilities :)

Reply:
renamed.